databricks/koalas/indexes.py

```diff
@@ -961,14 +961,16 @@ def _validate_index_level(self, level):
                 "Requested level ({}) does not match index name ({})".format(level, self.name)
             )
 
-    def copy(self, name=None):
+    def copy(self, name=None, deep=None):
```

Comment:
`MultiIndex` also has its own one. Could you address for it too?

Reply:
Oh, I missed it. I'm gonna address it, Thanks !